### PR TITLE
Add formatters and docstrings

### PR DESCRIPTION
**Pull request by alexsavio**

### CODE REVIEW

**Comment by sleistner** [comment]
still WIP?

File: lambda_handlers/handlers/event_handler.py [hunk 1]
```diff
@@ -27,15 +32,18 @@ class EventHandler(LambdaHandler, ValidationMixin, FormattingMixin):
...
+        self._input_format: Format = input_format or formatters.input_format.json
+        self._output_format: Format = output_format or formatters.output_format.json
 
     @property
     def validator(self) -> Validator:
+        """The event and result validator."""
         return self._validator
 
     def before(self, event, context):
+        """Event method called before the handler. It formats and validates `event`."""
```
Comment (sleistner) [inline lambda_handlers/handlers/event_handler.py]: "Event hook" maybe?

File: lambda_handlers/handlers/mixins/formatting_mixin.py [hunk 2]
```diff
@@ -1,7 +1,13 @@
+"""Formatting mixin."""
+
+
 class FormattingMixin:
+    """A mixin to user Formatters."""
```
Comment (sleistner) [inline lambda_handlers/handlers/mixins/formatting_mixin.py]: user Formatters?

File: lambda_handlers/handlers/tests/test_http_handler.py [hunk 3]
```diff
@@ -32,32 +33,33 @@ def test_handler_response(self, handler):
...
             'Access-Control-Allow-Origin': '*',
         }
 
 
-class PipeFormatter:
-    @staticmethod
-    def parse(content):
-        items = content.split('|')
-        return dict(zip(items[::2], items[1::2]))
+@format('application/piped')
```
Comment (sleistner) [inline lambda_handlers/handlers/tests/test_http_handler.py]: `'application/text+piped'` :)

File: lambda_handlers/handlers/event_handler.py (hunk 1 above)
Comment (alexsavio) [reply lambda_handlers/handlers/event_handler.py]: done

File: lambda_handlers/handlers/mixins/formatting_mixin.py (hunk 2 above)
Comment (alexsavio) [reply lambda_handlers/handlers/mixins/formatting_mixin.py]: fixed

File: lambda_handlers/handlers/tests/test_http_handler.py (hunk 3 above)
Comment (alexsavio) [reply lambda_handlers/handlers/tests/test_http_handler.py]: alright, changed

**Pull request merged**